src/notifications/emails.py

```
@@ -64,9 +73,13 @@
             await smtp.quit()
         except aiosmtplib.SMTPException as error:
             logging.error(f"Failed to send email to {recipient}: {error}")
-            raise BaseEmailError(f"Failed to send email to {recipient}: {error}")
+            raise BaseEmailError(
+                f"Failed to send email to {recipient}: {error}"
+            )
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)